tools/new_minor_OCP_version.py

```diff
@@ -209,21 +273,72 @@ def commit_and_push_version_update_changes(new_version, message_prefix):
     subprocess.check_output("git push origin HEAD:{}".format(branch), cwd="assisted-service", shell=True)
     return branch
 
+def commit_and_push_version_update_changes_app_interface(gitlab_key, fork, new_version, message_prefix):
+    def cmd(*args, **kwargs):
```

Comment:
You seem to run a similar command in `clone_app_interface` consider moving this function out.